src/components/Search/SearchRouter/SearchRouterList.tsx

```diff
@@ -411,9 +417,10 @@ function SearchRouterList(
     sections.push({title: translate('search.recentChats'), data: styledRecentReports});
 
     if (autocompleteSuggestions.length > 0) {
-        const autocompleteData = autocompleteSuggestions.map(({filterKey, text, autocompleteID}) => {
+        const autocompleteData = autocompleteSuggestions.map(({filterKey, text, autocompleteID, substitutionMapKey}) => {
             return {
                 text: getSubstitutionMapKey(filterKey, text),
+                mapKey: substitutionMapKey ? getSubstitutionMapKey(substitutionMapKey, text) : undefined,
```

Comment:
Can we please use the same name `substitutionMapKey` OR `mapKey` in both places?
I don't have a preference, I just want one name instead of two